wire-ios-data-model/Tests/MLS/ProteusToMLSMigrationCoordinatorTests.swift

```diff
@@ -60,6 +66,55 @@ class ProteusToMLSMigrationCoordinatorTests: ZMBaseManagedObjectTest {
         super.tearDown()
     }
 
+    // MARK: - Tests
+
+    func testMigrateOrJoinGroupConversations_CallsJoinGroupForNewConversations() async throws {
+        // GIVEN
+        let selfUser = ZMUser.selfUser(in: syncMOC)
+        selfUser.teamIdentifier = UUID()
+
+        let conversation = ZMConversation.insertNewObject(in: syncMOC)
+        conversation.teamRemoteIdentifier = selfUser.teamIdentifier
+        conversation.conversationType = .group
+        conversation.messageProtocol = .mixed
+        conversation.mlsGroupID = .random()
+
```

Comment:
you need to also that on the synMoc queue so await perform....

Reply:
Done!